src/services/api/currencies/currenciesApi.ts

```diff
@@ -0,0 +1,46 @@
+import axios from "axios";
+
+import { getCorrectBaseUrl } from "../../../utils/baseUrlConverter";
+
+interface ICurrencyItem {
+	title: string;
+	value: number;
+}
+
+export interface ICurrencyRates {
+	dollar: number;
+	euro: number;
+	crypto: number;
+}
+
+export const fetchCurrencyRates = async (): Promise<ICurrencyRates> => {
+	const baseUrl = getCorrectBaseUrl();
+
+	try {
+		const response = await axios.get<ICurrencyItem[]>(`${baseUrl}/currency`, {
```

Comment:
${baseUrl}/${currencyEndpoint}

can you create separate endpoint constant for this line?